react/ui-components/src/hoc/InboxSearchComposer.js

```
@@ -17,177 +17,200 @@ import HeaderComponent from "../atoms/HeaderComponent";
+  //         }
+  //     };
+  // }, [location]);
+
+  const configModule = Digit?.Customizations?.[apiDetails?.masterName]?.[apiDetails?.moduleName]
+  const updatedReqCriteria = configModule?.preProcess ? configModule?.preProcess(requestCriteria, configs.additionalDetails) : requestCriteria
+
+  if (configs?.customHookName) {
+    var { isLoading, data, revalidate, isFetching, refetch, error } = eval(`Digit.Hooks.${configs.customHookName}(updatedReqCriteria)`);
+  }
+  else {
+    var { isLoading, data, revalidate, isFetching, error, refetch } = Digit.Hooks.useCustomAPIHook(updatedReqCriteria);
```

Biome (1.9.4)

[error] Shouldn't redeclare 'isLoading'. Consider to delete it or rename it.

'isLoading' is defined here:


(lint/suspicious/noRedeclare)